images/linux/scripts/helpers/install.sh

```diff
@@ -95,3 +95,68 @@ get_github_package_download_url() {
     fi
     echo $downloadUrl
 }
+
+get_github_package_hash() {
+    local repo_owner=$1
+    local repo_name=$2
+    local file_name=$3
+    local url=$4
+    local version=${5:-"latest"}
+    local prerelease=${6:-false}
+    local delimiter=${8:-'|'}
```

Comment:
Did you skip 7th argument intentionally?
```suggestion
    local prerelease=${6:-false}
    local search_in_count=${7:-100}
    local delimiter=${8:-'|'}
```